app/services/container_dashboard_service.rb

```diff
@@ -99,9 +99,14 @@ def providers
   end
 
   def alerts
-    provider_ids = ManageIQ::Providers::ContainerManager.all.pluck(:id)
-    relation = @ems ? @ems.miq_alert_statuses : MiqAlertStatus.where(:ems_id => provider_ids)
-    alerts_status = relation.present? ? relation.where(:resolved => [false, nil]).group(:severity).count.values_at('error', 'warning') : [nil, nil]
+    providers = @ems.present? ? [@ems] : ManageIQ::Providers::ContainerManager.includes(:authentications)
+
+    has_alerts = providers.any? do |prov|
```

Comment:
can we rename `has_alerts` to `has_monitoring` please
Seeing has_alerts i'm thinking miq_alert_statuses